.github/workflows/check-lambda-js.yml

```diff
@@ -10,15 +10,16 @@ on:
       - main
       - development
       - ia-web-development
+      - ia-development
     paths:
-      - backend/compact-connect/lambdas/cloudfront-csp/**
+      - backend/compact-connect/lambdas/nodejs/**
 
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
```

Comment:
# Accessibility: Keyboard Navigation

When I select "Add state/jurisdiction", a new row appears but no item is selected for input. When I "TAB", it proceeds to the "Cancel" button, instead of the "State/Jurisdiction" dropdown, as I would expect.

<img width="847" alt="image" src="https://github.com/user-attachments/assets/f5228b1f-bd4e-42fd-ab6d-d8ce0c8f112e" />

Reply:
Do you mind creating a ticket for this one?